DistributedCollective/Sovryn-frontend
# Feat/perpetuals account

## QA:

Note: PR uses different BTCs for deposits and withdrawals than perpetuals trading, so you will see 2 different balances for now.
You can check balance only in withdrawal screen, balance visible in perpetuals page is not related to fast-btc deposits and withdrawals at this moment.

### Deposit
- open `/perpetual` page
- click "view account" (above place new order form)
- click "deposit"
- click "generate deposit address"
- send testnet BTC to address generated
- verify you received BTC  (may take some time to complete)

### Withdraw
- open `/perpetual` page
- click "view account" (above place new order form)
- click "withdraw"
- following withdrawal instructions and will in forms
- confirm transactions when asked (will be asked for 2 confirmations for the first time using withdrawal, otherwise you will be asked only once)
- verify you received BTC in your BTC testnet wallet (may take some time to complete)
